colrev/ops/pages/synthesizedrecords.py

```diff
@@ -0,0 +1,259 @@
+#! /usr/bin/env python3
+"""dashboard table and graphs for synthesized records"""
+from __future__ import annotations
+
+import bibtexparser
+import dash
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from dash import callback
+from dash import dash_table
+from dash import dcc
+from dash import html
+from dash import Input
+from dash import Output
+
+dash.register_page(__name__)
+
+
+with open(
```

Comment:
To load the records, the `review_manager.dataset.load_records_dict()` would be preferred (to keep the file handling in one place). To get a pandas df, the [from_records](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.from_records.html) method could be used (providing `records.values()` as the argument).